ingeniamc/k2-base-camp
# DR0053A-574 Update deps & fix scan

### Description

When connecting to the drive, a lot of error messages appear in the terminal and the data is not correctly (or not at all) plotted in the graphs.
It seems like the root cause of these issues is WinPcap - uninstalling it from the system and instead using Npcap fixed these problems.
This pull request was now highjacked to update the library dependencies and make a fixes related to connecting to the drive:
When the program was started before the drive was switched on, the scan would fail as the interface was not selected correctly. This should now no longer occur.

### Test

1. Start the program
2. Switch the drive on
3. Scan using the corresponding interface (this should be successful)
4. Select a dictionary and connect to the drive
5. Enable one or both motors and play with the controls
6. The graphs should correctly reflect the movements of the axes
